tests/integration/hosts/maya/input/startup/userSetup.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import sys
+
+from maya import cmds
+
+
+def setup_pyblish_logging():
+    log = logging.getLogger("pyblish")
+    hnd = logging.StreamHandler(sys.stdout)
+    fmt = logging.Formatter(
+        "pyblish (%(levelname)s) (line: %(lineno)d) %(name)s:"
+        "\n%(message)s"
+    )
+    hnd.setFormatter(fmt)
+    log.addHandler(hnd)
+
+
+def main():
+    cmds.evalDeferred("setup_pyblish_logging()", evaluateNext=True)
+    cmds.evalDeferred(
+        "import pyblish.util;pyblish.util.publish()", lowestPriority=True
+    )
```

Comment:
I feel like _if_ this is something we want during tests that this makes much more sense to be part of a global test-suite setup logic then being part of Maya specific startup code? The tests themselves run in Maya right, why can't the `setup` code set this up?

Reply:
Usually when running tests it triggers through something like a `unittest` runner or alike. These usually have a `setup` and `teardown` intended for this exact behavior.

E.g. the example test on [this page](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp)
```python
import unittest

class WidgetTestCase(unittest.TestCase):
    def setUp(self):
        self.widget = Widget('The widget')

    def tearDown(self):
        self.widget.dispose()
```

If we don't have that yet for running `HostTests` I feel we're better off developing _that_ explicitly so we can do so for all hosts than this _hack_ workaround.

Reply:
Will leave this up to @kalisp, who originally setup the framework.